src/Debugger/Impl/project.json

```diff
@@ -7,7 +7,7 @@
     "Microsoft.VisualStudio.SDK.EmbedInteropTypes": "15.0.10",
     "Microsoft.VisualStudio.Shell.15.0": "15.0.26606",
     "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319",
-    "Microsoft.VisualStudio.Text.Data": "15.0.26703.11111"
```

Comment:
Do key shortcuts work in the editor? I.e. Backspace for example. I remember that was the issue we've reverted for.

Reply:
Works fine with 15.5